test-common/src/main/resources/Dockerfile.local

```diff
@@ -1,4 +1,4 @@
-FROM    java:openjdk-8-jre-alpine
```

Comment:
I remember having recently some problems with using JRE image, so placed JDK here.

Reply:
In the Dockerfile for the server, we use adoptopenjdk/openjdk11:x86_64-alpine-jdk-11.0.16.1_1 as base image, maybe it would be a good idea to use the same for testing